### PR TITLE
Custom element showcase

### PR DESCRIPTION
**Pull request by maier49**
Adds a project that showcases widgets exported as custom elements.

### CODE REVIEW

File: custom-element-showcase/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "@dojo/cli-test-intern": "~0.2.0",
+    "@dojo/interfaces": "~0.1.0",
+    "@dojo/loader": "~0.1.0",
+    "@dojo/test-extras": "~0.2.1",
+    "@types/chai": "~3.4.0",
+    "@types/glob": "~5.0.0",
+    "@types/grunt": "~0.4.0",
+    "@types/node": "^6.0.46",
+    "intern": "4.0.2",
+    "typescript": "~2.4.1",
```
Comment (agubler) [inline custom-element-showcase/package.json]: reckon we could go 2.6.1?

File: custom-element-showcase/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "dojo2-custom-element-showcase",
+  "version": "1.0.0",
+  "dependencies": {
+    "@dojo/core": "~0.2.1",
+    "@dojo/has": "~0.1.0",
+    "@dojo/routing": "~0.2.0",
```
Comment (agubler) [inline custom-element-showcase/package.json]: does this actually use routing?

File: custom-element-showcase/src/index.html [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+        </div>
+        <h2>Split Pane</h2>
+        <dojo-split-pane direction="0" size="300"></dojo-split-pane>
+        <h2>Calendar</h2>
+        <dojo-calendar></dojo-calendar>
+        <h2>Accordion Pane</h2>
+    </div>
+    <script>
+        function initializeSplitPane() {
+			var splitPane = document.querySelector('dojo-split-pane');
```
Comment (agubler) [inline custom-element-showcase/src/index.html]: indentation?

File: custom-element-showcase/src/index.html [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+			splitPane.trailing = 'Trailing Content';
+        }
+
+        function initializeTooltip() {
+        	var tooltip = document.querySelector('dojo-tooltip');
+        	tooltip.content = 'Tooltip content';
+        }
+
+        function createAccordionPane() {
+			var parentDiv = document.createElement('div');
```
Comment (agubler) [inline custom-element-showcase/src/index.html]: indentation?

**Comment by maier49** [comment]
Updated to use TS 2.6.1, removed @dojo/router dependency, and fixed formatting inconsistencies

**Comment by agubler** [comment]
Closing, the @dojo/widgets custom elements are now published as of 0.5.0 and an updated showcase will be added soon!

**Pull request closed without merging**